.github/workflows/ci_test-conda.yml

```diff
@@ -1,4 +1,4 @@
-name: Test
+name: Test with Conda
```

Comment:
It's just test because the job name is already appended

`Test / conda`

vs 

`Test with Conda / conda`

Reply:
yes, bu the workflows are duplicate

Reply:
The current structure easily differentiates all tests jobs.

Also reduces the clutter

Reply:
can you explain how it reduces clutter?

Reply:
not sure who did it but do see it much useful...
![image](https://user-images.githubusercontent.com/6035284/148422184-ec02a566-12c5-4f4a-9bcf-d3dafce6937c.png)

Reply:
Can you drop the test name changes from this PR and open an issue about it?

Let's keep this PR focused just to the concurrency feature

Reply:
without having a unique name per workflow (as the workflow name is used in group creation) does not make much sense... Did we have an issue with renaming all to `Test`?

Reply:
the workflow name sets it as queue so having only one means that later on master all these will run in a sequence not parallel

Reply:
> Did we have an issue with renaming all to Test?

The job view in the PR looked cluttered.

> found your initial change in #10075 and it seems you did not border with the issue discussion either :D

You approved that PR (including the job name change). If you had requested me to open an issue for further discussion, I would have gladly done so.

I am requesting you to do it because I'm sure we can find a middle ground between both options, just needs previous discussion.

Reply:
> You approved that PR (including the job name change)

sorry, I do not see it as approved
![image](https://user-images.githubusercontent.com/6035284/149538464-622ecd23-c2f2-4dd3-bca7-bb7b843ddd1b.png)

Reply:
You initially requested changes, and later approved the PR.
<img width="1379" alt="Screenshot 2022-01-14 at 16 49 59" src="https://user-images.githubusercontent.com/11067892/149544554-03857143-393b-4aeb-84ad-f13edd45adae.png">
Clicking on "View changes" shows that the PR already included the job name change at the time of the approval.

Reply:
> You initially requested changes, and later approved the PR.

and later as you can also see withdrew my approval as I did not agree with but didn't want to play this game :disappointed: